packages/react-native/Libraries/Components/View/View.js

```diff
@@ -14,7 +14,10 @@ import TextAncestor from '../../Text/TextAncestor';
 import ViewNativeComponent from './ViewNativeComponent';
 import * as React from 'react';
 
```

Comment:
We should also switch to destructuring these imports as recommended by React docs.

```suggestion
import * as React from 'react';
import {use} from 'react';

```

Reply:
Was matching the style in LogBox